packages/manager/modules/pci/src/projects/project/notebooks/add/add.html

```diff
@@ -1,13 +1,136 @@
+            data-display="$ctrl.displayNotebookFramework"
+            data-notebook="$ctrl.notebook"
+            data-frameworks="$ctrl.frameworks"></notebook-frameworks-list>
+    </oui-step-form>
+
+    <!--Notebook Privacy Settings-->
+    <oui-step-form
+        name="notebook_private_settings"
+        data-header="{{:: 'pci_notebook_add_privacy_settings_title' | translate }}"
+        data-on-focus="$ctrl.displayNotebookPrivacySettings = true"
+        data-on-submit="$ctrl.displayNotebookPrivacySettings = false"
+        data-navigation="$ctrl.notebook.selected.privacy"
+    >
+        <notebook-privacy-settings
+            data-display="$ctrl.displayNotebookPrivacySettings"
+            data-notebook="$ctrl.notebook"></notebook-privacy-settings>
+    </oui-step-form>
+
+    <!--Notebook Datacenter Location-->
+    <oui-step-form
+        name="notebook_datacenter_location"
+        data-header="{{:: 'pci_notebook_add_privacy_datacenter_location_title' | translate }}"
+        data-on-focus="$ctrl.displayNotebookDatacenterLocation = true"
+        data-on-submit="$ctrl.onLocationStepSubmit()"
+        data-navigation="$ctrl.notebook.selected.region"
+    >
+        <pci-project-regions-list
+            data-regions="$ctrl.regions"
+            data-selected-region="$ctrl.notebook.selected.region"
+            data-display-selected-region="$ctrl.displaySelectedRegion"
+        ></pci-project-regions-list>
+    </oui-step-form>
+
+    <!--Notebook Resources-->
+    <oui-step-form
+        name="notebook_resources"
+        data-header="{{:: 'pci_notebook_add_resources_title' | translate }}"
+        data-on-focus="$ctrl.displayNotebookResources = true"
+        data-on-submit="$ctrl.displayNotebookResources = false"
+        data-navigation="$ctrl.notebook.selected.resource.usage && $ctrl.notebook.selected.resource.flavor"
+    >
+        <notebook-resources
+            data-display="$ctrl.displayNotebookResources"
+            data-notebook="$ctrl.notebook"
+            data-flavors="$ctrl.flavors"></notebook-resources>
+    </oui-step-form>
+
+    <!--Notebook Attach Container and Repository-->
+    <oui-step-form
+        name="notebook_attach"
+        data-header="{{:: 'pci_notebook_add_attach_title' | translate }}"
+        data-on-focus="$ctrl.displayNotebookAttach = true"
+        data-on-submit="$ctrl.displayNotebookAttach = false"
+        data-prevent-next="true"
+    >
+        <notebook-attach
+            data-display="$ctrl.displayNotebookAttach"
+            data-notebook="$ctrl.notebook"
+            data-volumes="$ctrl.volumes"></notebook-attach>
+    </oui-step-form>
+
+    <!--Notebook Review-->
+    <oui-step-form
+        name="notebook_attach"
+        data-header="{{:: 'pci_notebook_add_review_title' | translate }}"
+        data-on-focus="$ctrl.displayNotebookReview = true"
+        data-on-submit="$ctrl.displayNotebookReview = false"
+        data-editable="!$ctrl.isAdding"
+    >
+        <notebook-review
```

Comment:
👍

Reply:
Yes we have a dedicated one for the content.